modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/api.mustache

```diff
@@ -172,18 +172,39 @@ public class {{classname}} {
         }
 
         {{/queryParams}}
+        {{#constantParams}}
+        {{#isQueryParam}}
+        // Set client side default value of Query Param "{{baseName}}".
+        localVarCollectionQueryParams.add(new Pair("{{baseName}}", {{#_enum}}"{{.}}"{{/_enum}}));
```

Comment:
```
{{#_enum}}"{{.}}"{{/_enum}}
```

please use the following instead (original value instead of HTML escaped value)
```
{{#_enum}}"{{{.}}}"{{/_enum}}
```